### ISSUE TITLE
"Run All" keyboard shortcut `Ctrl+Alt+R` not working in R Markdown documents

### ISSUE THREAD

**Issue by jhuntergit** [rstudio/rstudio#5632]
### System details

    RStudio Edition : Desktop Preview Version 1.2.1578
    OS Version      : Windows 10 Pro
    R Version       : 3.6.1

### Steps to reproduce the problem

1. Create a new default R Markdown document in R Studio via `File > New File > R Markdown`
2. Save the R Markdown document
3. Place the cursor on line 11 (in between `## R Markdown` and `This is an R Markdown ...`)
4. Verify that keyboard shortcuts are working by initiating "Run Next Chunk" with `Ctrl+Alt+N`.
5. If results from `summary(cars)` appear in your console proceed
6. Place the cursor on line 11 (in between `## R Markdown` and `This is an R Markdown ...`)
7. Verify "Run All" works in R studio by clicking on `Run > Run All`. 
8. Again, place the cursor on line 11 (in between `## R Markdown` and `This is an R Markdown ...`)
9. Test the keyboard shortcut for "Run All" by pressing `Ctrl+Alt+R` simultaneously.
10. If nothing happens, as is the case on my computer, reply to this thread. 

Thank you for taking the time to read this.

**Comment by dfalty** [comment]
I can't reproduce this on the preview build 1.2.5018 but on Mac. Does it work for you on that build? If not, it may be a windows problem but my Windows VM is acting up atm so I can't check.

**Comment by jhuntergit** [comment]
Doesn't happen on Ubuntu 18.04 either. Looks to be a Windows specific issue.

**Comment by ronblum** [comment]
@jhuntergit Have you by chance tried out the 1.3 Middle Mist preview https://rstudio.com/products/rstudio/download/preview/ ? I tried it out on Windows 10 and it runs properly,

**Comment by jhuntergit** [comment]
Just tried it out on 1.3 Middle Mist and am still having the same issue. Might just be isolated to this Win10 box, don't have others to try. All other shortcuts appear to be working fine. Feel free to close, it may be unlikely we're able to get a repro. Thanks for trying.

**Comment by ronblum** [comment]
@jhuntergit Thank you for checking! As a workaround, what happens if you remap the keyboard shortcut?
- Tools -> Modify Keyboard Shortcuts... -> Run All Code in Current Source File

**Comment by jhuntergit** [comment]
@ronblum That's a great idea. I changed the shortcut to `Ctrl+Shift+Alt+R` and it works fine. I then tried to change it back to the standard `Ctrl+Alt+R` and it would not take at all in `Tools > Modify Keyboard Shortcuts... > Run All Code...`. I keep slamming `Ctrl+Alt+R` and it won't populate the blank "cell". 

I then opened Windows 10 On-Screen Keyboard and started mashing `Ctrl+Alt+R` and could see `Ctrl` and `Alt` lighting up, but not the `R`. I then started tapping all the alphabet keys and none of them seem to light up on this On-Screen Keyboard, so this test has limited utility. Hope that helps. I could probably download another on-screen key program, but I'm happy with your solution, and figure this might just be a Win10 OS issue (or not).
